elements/campaign-display/components/campaign-display-image.js

```diff
@@ -3,13 +3,14 @@ import React, { PropTypes } from 'react';
 export default function CampaignDisplayImage (props) {
   return (
     <div className='campaign-display-image'>
-      <a href={props.url}>
-        <img src={props.image} />
+      <a href={props.clickthrough_url}>
```

Comment:
Actually, nevermind, if there's a problem with the analytics manager here, we should fix it in the analytics manager and keep this clean.